### ISSUE TITLE
Sphinx-generated wiki build issues

### ISSUE THREAD

**Issue by oliver-leung** [AguaClara-Reach/aguaclara#108]
The `aguaclara` wiki is looking a bit disorganized - I'm thinking that we should move towards Sphinx auto-generated documentation, like the [textbook](https://aguaclara.github.io/Textbook/) and [`onshapepy`](https://aguaclara.github.io/onshapepy/).

There's already a [documentation website](https://aguaclara.github.io/aguaclara/#) set up for the repository, which automatically updates when our Travis build passes. I think we should remove the current wiki, move all relevant documentation to the Sphinx-generated site, then link to the Sphinx docs from `README.md`.

These Sphinx docs should contain:
- Code documentation for all classes, functions, and constants
- Instructions on using the package
- Instructions on developing the package

AIDE Design is currently working on the `aguaclara.design` code for the flocculator - we'll try to implement Sphinx documentation for that when we merge our changes with `master`.

We should also rewrite `README.md` to only contain quick installation instructions and a link to the Sphinx docs once they're made.

**Comment by oliver-leung** [comment]
Hey @HannahSi , are you planning on writing documentation on how to use the package's graphing functionality? If so, you should consider doing it on the Sphinx-generated wiki!

If you'd like, we can have a meeting this week or the week after to go over some things I've figured out about Sphinx.

**Comment by HannahSi** [comment]
Hi Oliver, I think moving the wiki to a Sphinx documentation site is a good idea! Yes, I do plan on writing documentation for functions I added to the package and would be happy to talk about generating it with Sphinx. I'm available 3-5pm today and Monday, after 4:30 this Thursday. What time works for you?

**Comment by oliver-leung** [comment]
Let's meet next Monday at 3pm, talk to you then!

**Comment by HannahSi** [comment]
The Sphinx documentation is still not building, and it seems that the docs/build folder is not included in the master branch. @oliver-leung Do you have a folder locally that you can add? And should we delete docs/_build (and also build/?) from the .gitignore?

**Comment by oliver-leung** [comment]
Sorry, just saw this! Travis is supposed to build the finalized HTML files that go into the wiki, so we avoid adding build files to the repository that would clutter it up.

Those build files are supposed to be added to the `gh-pages` branch, although it seems that they're still not appearing there.

**Comment by oliver-leung** [comment]
Hmm... It seems that https://aguaclara.github.io/aguaclara/ still isn't building right. I think it might be an issue with our Travis configuration, so I'll try to get that fixed for release v0.1.0!

**Comment by HannahSi** [comment]
Reviving this issue... it looks like the wiki page wasn't building correctly because the `local-dir` configuration in the .travis.yml was set to `docs/build` instead of `docs/build/html`. (I noticed this difference with the working [`aguaclara_tutorial` repository](https://github.com/AguaClara/aguaclara_tutorial).) I changed this in release tag 0.0.24 and the website is building now! 

I didn't realize that Travis could deploy to gh-pages from release tags, not just builds of the master branch, so pull request #197 is still open. Don't worry about reviewing right away since we're on break, but feedback on the other changes later would be appreciated!
